app/assets/javascripts/listings/templates/listings.html.slim

```diff
@@ -1,9 +1,52 @@
-section.sub-bar.bg-mist
+header.lead-header.bg-attention-tint
   .row
-    h3.sub-bar_header.epsilon
-      | Estimate Eligibility
-    a.button--start.button.tiny ui-sref="dahlia.eligibility-estimator"
-      | {{ hasEligibilityFilters() ? 'Edit' : 'Get Started' }}
+    .large-12.columns
+      h1.lead-title.t-beta Below Market Rate Rentals
+      .medium-8.columns.no-padding
+        p.t-gamma.t-light.show-for-medium-up Units in new developments priced at a “below market rate” are affordable to lower or middle income households.
+        p.no-margin
+          a.button.small.attention ui-sref="dahlia.eligibility-estimator"
+            | {{ hasEligibilityFilters() ? 'Edit Eligibility Estimate' : 'Estimate Eligibility' }}
+
+//// Matches results bar
```

Comment:
That code represents the markup patterns for this story:
https://www.pivotaltracker.com/n/projects/1405352/stories/107537070

So, I say no.